src/features/home/components/SettingsList.tsx

```diff
@@ -0,0 +1,100 @@
+import { FC, useState } from 'react';
+import {
+  Box,
+  Button,
+  Divider,
+  FormControl,
+  InputLabel,
+  MenuItem,
+  Select,
+} from '@mui/material';
+
+import useUserMutations from '../hooks/useUserMutations';
+import { ZetkinUser } from 'utils/types/zetkin';
+import ZUICard from 'zui/ZUICard';
+import messageIds from '../l10n/messageIds';
+import { Msg, useMessages } from 'core/i18n';
+
+export type ZetkinLanguage = 'en' | 'sv' | 'da' | 'nn' | 'de' | null;
```

Comment:
I suspect this type is handy also in other places, and could probably live in the Zetkin types instead of in this component?

Reply:
It could be, but it hasn't been used anywhere else yet, so I don't mind keeping it here until we need it again, if that's ok?